docs/website/blog/2024-03-13-smart-dashboarding-tools.md

```diff
@@ -0,0 +1,141 @@
+---
+slug: smart-dashboarding-tools
+title: "What is so smart about smart dashboarding tools?"
+image: https://storage.googleapis.com/dlt-blog-images/smart-dashboarding-question-example.png
+authors:
+  name: Hiba Jamal
+  title: Data Science intern at dltHub
+  url: https://github.com/hibajamal
+  image_url: https://avatars.githubusercontent.com/u/35984866?v=4
+tags: [dashboarding tools, LLMs, AI driven analytics]
+---
+
+<aside>
+💡 TL;DR: Exploring the theme of "AI replacing jobs," we examine ThoughtSpot's LLM analyst, Sage. This article assesses Sage's application in different analytics techniques, the importance of the data model it analyzes, and dlt's supportive role in this context.
+</aside>
+
+## The advent of ChatGPT...
+...sparked widespread speculation about the future of *many* professions, analytics included. Now, over a year later, and with an array of GPT and LLM-powered analytics tools at our disposal, we're in a stronger position to assess their intelligence and capabilities.
+
+In this article, we explore ThoughtSpot, known for its simplicity and strong data democracy practices. However, our focus narrows to Sage, its LLM assistant, and examining how it can empower or replace analytics techniques carried out by different analysts.
+
+## Analysts: Fallen victims of AI or not?
+
+The data analyst's role encompasses various job descriptions – from data science to dashboarding, data pipeline management, and even ML engineering. However, for this blog, we'll focus on the four key analytics components or techniques that help a company achieve its goals, as outlined by [Garter](https://www.gartner.com/en/topics/data-and-analytics#q8).
```

Comment:
typo:
 as outlined by [GartNer]

Reply:
![image](https://github.com/dlt-hub/dlt/assets/35984866/bdf3ac4b-5da8-4c43-9bc2-e9d7bf8e3d85)

i only saw this logo throughout their website.. should i still change it to [GartNer]?

Reply:
no, no, I mean you wrote it here as  `Garter`, missed the 'n' letter, change it to [Gartner]